### PR TITLE
GUACAMOLE-1611: Add new French translations.

### PR DESCRIPTION
**Pull request by corentin-soriano**
Added the latest missing translations for 1.6.0.

### CODE REVIEW

File: extensions/guacamole-auth-totp/src/main/resources/translations/fr.json [hunk 1]
```diff
@@ -29,6 +29,24 @@
 
         "SECTION_HEADER_DETAILS" : "Détails:"
 
+    },
+
+    "USER_ATTRIBUTES" : {
+        
+        "FIELD_HEADER_GUAC_TOTP_DISABLED"      : "Désactiver TOTP :",
+        "FIELD_HEADER_GUAC_TOTP_KEY_GENERATED" : "Clé secrète générée :",
+        "FIELD_HEADER_GUAC_TOTP_KEY_CONFIRMED" : "Appareil d'authentification confirmé :",
```
Comment (necouchman) [inline extensions/guacamole-auth-totp/src/main/resources/translations/fr.json:38]: I think there's an extra space on each of these lines between the text and the ":".

Comment (corentin-soriano) [reply extensions/guacamole-auth-totp/src/main/resources/translations/fr.json]: Extra spaces removed!

File: guacamole/src/main/frontend/src/translations/fr.json [hunk 2]
```diff
@@ -633,17 +633,20 @@
...
         "FIELD_HEADER_SFTP_PASSWORD"    : "Mot de passe:",
         "FIELD_HEADER_SFTP_PORT"        : "Port:",
         "FIELD_HEADER_SFTP_PRIVATE_KEY" : "Clé privée:",
+        "FIELD_HEADER_SFTP_PUBLIC_KEY"  : "Clé publique:",
         "FIELD_HEADER_SFTP_ROOT_DIRECTORY" : "Dossier racine de l'explorateur de fichier:",
-        "FIELD_HEADER_SFTP_DISABLE_UPLOAD"        : "Désactiver l'envoi de fichier:",
+        "FIELD_HEADER_SFTP_TIMEOUT"        : "Délai d'expiration de la connexion SFTP:",
         "FIELD_HEADER_SFTP_USERNAME"    : "Identifiant:",
         "FIELD_HEADER_STATIC_CHANNELS"  : "Noms des canaux statiques:",
+        "FIELD_HEADER_TIMEOUT"         : "Délai d'expiration de la connexion",
```
Comment (jmuehlner) [inline guacamole/src/main/frontend/src/translations/fr.json:649]: It looks like there's one missing space here.

**Comment by jmuehlner** [comment]
LGTM aside from one very minor spacing issue.

File: guacamole/src/main/frontend/src/translations/fr.json (hunk 2 above)
Comment (corentin-soriano) [reply guacamole/src/main/frontend/src/translations/fr.json]: Added!

File: guacamole/src/main/frontend/src/translations/fr.json [hunk 3]
```diff
@@ -633,17 +633,20 @@
         "FIELD_HEADER_SERVER_LAYOUT"   : "Agencement clavier:",
         "FIELD_HEADER_SFTP_DIRECTORY"   : "Répertoire d'upload par défaut:",
         "FIELD_HEADER_SFTP_DISABLE_DOWNLOAD" : "Désactiver le téléchargement de fichier:",
+        "FIELD_HEADER_SFTP_DISABLE_UPLOAD"        : "Désactiver l'envoi de fichier:",
```
Comment (necouchman) [inline guacamole/src/main/frontend/src/translations/fr.json:636]: Maybe line this up with the one above?

**Review by necouchman — CHANGES_REQUESTED** [review]
Sorry @corentin-soriano - Just one more change that I can't quite let go! :laughing:

File: guacamole/src/main/frontend/src/translations/fr.json (hunk 3 above)
Comment (corentin-soriano) [reply guacamole/src/main/frontend/src/translations/fr.json]: Changed!

**Review by necouchman — APPROVED** [review]

**Pull request merged**